ui-tests/test/contents.test.ts

```diff
@@ -27,6 +27,11 @@ test.use({
 
 test.describe('Contents Tests', () => {
   test.beforeEach(async ({ page }) => {
+    page.on('console', (message) => {
+      // eslint-disable-next-line no-console
+      console.log('CONSOLE MSG', message.text());
+    });
```

Comment:
Maybe we don't need this anymore if it was for debugging?

Otherwise such console logging should probably apply to all UI tests, not just for the contents?

Reply:
OK we can add it again globally later if needed.

Reply:
> apply to all UI tests

Being able to see `console.log`s in the test outputs is useful IMO, I always end up adding these three lines for debugging purpose.

I would be +1 for keeping that around.

Reply:
Let's track that in an issue?